sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/Base/AlgorithmEnums.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Azure.Security.KeyVault.Cryptography.Base
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    internal enum AlgorithmCategoryEnum
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        Symmetric,
+
+        /// <summary>
+        /// 
+        /// </summary>
+        Asymmetric,
+
+        /// <summary>
+        /// 
+        /// </summary>
+        UnSupported
+    }
+
+    /// <summary>
+    /// 
+    /// </summary>
+    public enum EncryptionAlgorithmKindEnum
```

Comment:
This should just be EncryptionAlgorithm

Reply:
EncryptionAlgorithm is vague and sounds like a type name that might have operations on it, it should at least be EncryptionAlgorithmKind